Web/Admin.UI/src/app/services/gateway/normalization/operation.service.ts

```
@@ -9,257 +9,280 @@ import {OperationType} from "../../../interfaces/normalization/operation-type-en
+        resourceType: string | null,
+        operationId: string | null,
+        includeDisabled: boolean | null,
+        vendorId: string | null,
+        sortBy: string | null,
+        sortOrder: 'ascending' | 'descending' | null,
+        pageSize: number,
+        pageNumber: number
+    ): Observable<IPagedOperationModel> {
+
+        //java based paging is zero based, so increment page number by 1
+        pageNumber = pageNumber + 1;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)